setup.sh

```diff
@@ -121,6 +124,16 @@ fail () {
   exit 1
 }
 
+check_apis_enabled () {
+  SERVICES=`gcloud services list --enabled`
```

Comment:
Up to you: `gcloud services list --enabled --format='value(config.title)'` will give you a more succinctly formatted list.